.gitattributes

```diff
@@ -0,0 +1,9 @@
+.editorconfig export-ignore
+.gitattributes export-ignore
+.github export-ignore
+.gitignore export-ignore
+.travis.yml export-ignore
+CHANGELOG.md export-ignore
+README.md export-ignore
```

Comment:
@sunaoka thnx for PR.
I think it will be better if `CHANGELOG.md` & `README.md` remains in the repository

Reply:
@alek13 thnx for review.
`CHANGELOG.md` and `README.md` now remain in the repository.